### DISCUSSION TITLE
UUIDv4

### DISCUSSION THREAD

**Discussion by icehitter** [OpenIDC/mod_auth_openidc#603] [Q&A]
We use apache 2.4.46.  We currently are only able to get UUIDv1 to generate.  We are using the mod_auth_openidc 1.8.10 module.  I have tried using different versions of the mod_auth_openidc module, but I am not able to get anything besides UUIDv1 to generate.  I looked at the documentation for 2.4.48, but I have not been able to find anything concerning UUID.  At this point, we are looking at going RedHat apache 2.4.6 to see if the their standard install provides for UUIDv4.  Can UUIDv4 be generated using apache 2.4.46 or apache 2.4.48?

**Comment by zandbelt** [comment]
please use a recent version of the module

**Reply by icehitter** [reply]
What version of the module should we be using?

**Reply by zandbelt** [reply]
see also https://github.com/zmartzone/mod_auth_openidc/issues/431#issuecomment-1312811669
